src/Metacello-Core/MetacelloAbstractPackageSpec.class.st

```diff
@@ -23,7 +23,7 @@ MetacelloAbstractPackageSpec >> addToMetacelloPackages: aMetacelloPackagesSpec [
 { #category : #querying }
 MetacelloAbstractPackageSpec >> answers [
 
-	answers == nil ifTrue: [ answers := #() ].
+	answers ifNil: [ answers := #() ].
 	^answers
```

Comment:
you could even move the return up and have everything in one line:

^answers ifNil: [ answers := #() ].

Reply:
Thx ! I just pushed the commit to have all in one line.